src/client/utils/browser.js

```diff
@@ -22,40 +22,56 @@ function calculateBrowserAndVersion (userAgent) {
     };
 }
 
+
+export var isIE11;
```

Comment:
@churkin I don't like what we turn pure utilities into the class object just because of tests, it's this kind of thing which nearly shouts in your face: "We need to rewrite those tests! They screw up our production code!"